test/moves/u_turn.test.ts

```diff
@@ -82,7 +82,7 @@ describe("Moves - U-turn", () => {
 
     // assert
     const playerPkm = game.scene.getPlayerPokemon()!;
-    expect(playerPkm.status?.effect).toEqual(StatusEffect.POISON);
+    expect(playerPkm.getStatusEffect()).toEqual(StatusEffect.POISON);
```

Comment:
```suggestion
    expect(playerPkm.getStatusEffect(true)).toEqual(StatusEffect.POISON);
```

Reply:
This should use `toBe`:
```suggestion
    expect(playerPkm.getStatusEffect()).toBe(StatusEffect.POISON);
```